src/zenml/constants.py

```diff
@@ -399,6 +399,11 @@ def handle_int_env_var(var: str, default: int = 0) -> int:
 PIPELINE_SPEC = "/pipeline-spec"
 PLUGIN_FLAVORS = "/plugin-flavors"
 PROJECTS = "/projects"
+PROMPTS = "/prompts"
+PROMPT_TEMPLATES = "/prompt_templates"
+PROMPT_EXPERIMENTS = "/prompt_experiments"
+PROMPT_EVALUATIONS = "/prompt_evaluations"
+PROMPT_DEPLOYMENTS = "/prompt_deployments"
```

Comment:
some of this relates to functionality not in this PR, so only keep what we're using